### PR TITLE
[Feature-8025][Document] Add example and notice about task type HTTP

### PR DESCRIPTION
**Pull request by QuakeWang**
close: https://github.com/apache/dolphinscheduler/issues/8025

### CODE REVIEW

File: docs/en-us/dev/user_doc/guide/task/http.md [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
+- **Number of failed retry attempts**: The number of times the task failed to be resubmitted.
+- **Failed retry interval**: The time, in cents, interval for resubmitting the task after a failed task.
+- **Delayed execution time**: the time, in cents, that a task is delayed in execution.
+- **Timeout alarm**: Check the timeout alarm and timeout failure. When the task exceeds the "timeout period", an alarm email will be sent and the task execution will fail.
+- **Request address**: HTTP request URL.
+- **Request type**: Support GET, POSt, HEAD, PUT, DELETE.
+- **Request parameters**: Support Parameter, Body, Headers.
+- **Verification conditions**: support default response code, custom response code, content included, content not included.
+- **Verification content**: When the verification condition selects a custom response code, the content contains, and the content does not contain, the verification content is required.
+- **Custom parameter**: It is a user-defined parameter of http part, which will replace the content with \${variable} in the script.
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/task/http.md:29]: Hi, I find out you change the content already changed by @Tianqi-Dotes, for example `\${variable}` already change  d to `${variable}`, Please copy the latest content and change

**Review by zhongjiajie — CHANGES_REQUESTED** [review]

File: docs/en-us/dev/user_doc/guide/task/http.md (hunk 1 above)
Comment (QuakeWang) [reply docs/en-us/dev/user_doc/guide/task/http.md]: > Hi, I find out you change the content already changed by @Tianqi-Dotes, for example `\${variable}` already change d to `${variable}`, Please copy the latest content and change

Done : )

**Comment by zhongjiajie** [comment]
Hi @QuakeWang, you should also check other content and make sure do not overwrite then content

**Comment by QuakeWang** [comment]
> Hi @QuakeWang, you should also check other content and make sure do not overwrite then content

Do you mean to detect overwriting in all task files?

**Comment by zhongjiajie** [comment]
Yeah, that's what I mean, forget me poor English

**Comment by QuakeWang** [comment]
> Yeah, that's what I mean, forget me poor English

OK, I got it.

**Comment by zhongjiajie** [comment]
I restart the failed test. When it ready to go, please mention me, thank

**Comment by QuakeWang** [comment]
> I restart the failed test. When it ready to go, please mention me, thank

Done

**Review by zhongjiajie — APPROVED** [review]
Great! thanks

**Pull request merged**